pulsar-testclient/src/main/java/org/apache/pulsar/proxy/socket/client/PerformanceClient.java

```diff
@@ -95,7 +95,10 @@
         @Parameter(names = { "-t", "--num-topic" }, description = "Number of topics", validateWith = PositiveNumberParameterValidator.class)
         public int numTopics = 1;
 
-        @Parameter(names = { "--auth_plugin" }, description = "Authentication plugin class name")
+        @Parameter(names = { "--auth_plugin" }, description = "Authentication plugin class name", hidden = true)
+        public String DEPRECATED_authPluginClassName;
```

Comment:
This is bad name for a Java field.

Please use Camel case

Reply:
Okay :)